lib/ReactViews/Map/Navigation/Items/MyLocation.ts

```diff
@@ -142,60 +151,47 @@ const MyLocation = createReactClass({
       const secureUrl = uri.protocol("https").toString();
       message = t("location.originError", { secureUrl: secureUrl });
     }
-    this.props.terria.error.raiseEvent(
+    this.terria.error.raiseEvent(
       new TerriaError({
         sender: this,
         title: t("location.errorGettingLocation"),
         message: message
       })
     );
-  },
+  }
 
   augmentedVirtualityEnabled() {
-    return (
-      defined(this.props.terria.augmentedVirtuality) &&
+    /* return (
+      isDefined(this.props.terria.augmentedVirtuality) &&
       this.props.terria.augmentedVirtuality.enabled
-    );
-  },
+    ); */
+    return false;
+  }
```

Comment:
Does this affect the augmentedVirtuality feature?

Reply:
Wasn't able to figure out from where is `augmentedVirtuality` started since it was missing from `Terria.ts` but now see that it is added from `AugmentedVirtualityTool`. Should port AugmentedVirtualityTool to typescript and register it as a new navigation element, probably can do this by the end of the week